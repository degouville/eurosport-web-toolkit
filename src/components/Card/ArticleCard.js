import React from 'react';
import { Card, CardHeader, CardContent, CardFooter } from './base/index';

const ArticleCard = () => (
  <Card>
    <CardHeader />
    <CardContent />
    <CardFooter />
  </Card>
);

ArticleCard.displayName = 'ArticleCard';

export default ArticleCard;
