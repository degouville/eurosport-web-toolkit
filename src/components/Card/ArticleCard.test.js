import React from 'react';
import { shallow } from 'enzyme';
import ArticleCard from './ArticleCard';

it('renders a ArticleCard', () => {
  expect(shallow(<ArticleCard />)).toMatchSnapshot();
});
