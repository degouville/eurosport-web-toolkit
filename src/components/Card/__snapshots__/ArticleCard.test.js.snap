// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a ArticleCard 1`] = `
<Card>
  <CardHeader />
  <CardContent />
  <CardFooter />
</Card>
`;
